tests/app/notifications/test_validators.py

```diff
@@ -276,16 +277,19 @@ def test_service_can_send_to_recipient_fails_when_ignoring_safelist(
 @pytest.mark.parametrize('key_type, error_message',
                          [('team', 'Can’t send to this recipient using a team-only API key'),
                           ('normal',
-                           "Can’t send to this recipient when service is in trial mode – see https://www.notifications.service.gov.uk/trial-mode")])  # noqa
-def test_service_can_send_to_recipient_fails_when_recipient_is_not_on_team(recipient, key_type, error_message,
-                                                                           notify_db, notify_db_session):
+                           "Can’t send to this recipient when service is in trial mode – see ")])  # noqa
+def test_service_can_send_to_recipient_fails_when_recipient_is_not_on_team(recipient: str,
+                                                                           key_type: str,
+                                                                           error_message: str,
+                                                                           notify_db: RoutingSQLAlchemy,
+                                                                           notify_db_session: RoutingSQLAlchemy):
```

Comment:
Would be a more common indentation
```suggestion
def test_service_can_send_to_recipient_fails_when_recipient_is_not_on_team(
    recipient: str,
    key_type: str,
    error_message: str,
    notify_db: RoutingSQLAlchemy,
    notify_db_session: RoutingSQLAlchemy
):
```

Reply:
formatting done